app/controllers/reward_controller.rb

```diff
@@ -20,4 +21,8 @@ def set_response
   def set_protocol_subscription
     @protocol_subscription = @response.protocol_subscription
   end
+
+  def use_mentor_layout
+    @use_mentor_layout = @response.protocol_subscription.person.mentor?
```

Comment:
Is dit niet gewoon `@use_mentor_layout = false`? Of kunnen mentoren nu ook bij de rewards page? Wat is er precies veranderd in de view van rewardpage dat we nu ineens deze `@use_mentor_layout` variabele nodig hebben trouwens?

Reply:
op false gezet. Een logo bovenaan.